action/protocol/staking/bucket_pool_test.go

```diff
@@ -66,8 +66,12 @@ func TestBucketPool(t *testing.T) {
 
 	pool, err := NewBucketPool(sm)
 	r.NoError(err)
-	r.Equal(big.NewInt(0), pool.Total())
-	r.EqualValues(0, pool.Count())
+	total, _, err := pool.Total(sm)
```

Comment:
how could we passing sm to both NewBucketPool and Total

Reply:
fixed